recipes/watchdog/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "watchdog" %}
+{% set version = "0.8.3" %}
+{% set hash_type = "md5" %}
+{% set hash = "bb16926bccc98eae2a04535e4512ddf1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/w/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
Please change to `https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz`.